io/plugins/eu.esdihumboldt.hale.io.jdbc.test/src/eu/esdihumboldt/hale/io/jdbc/test/DBImageParameters.java

```diff
@@ -0,0 +1,96 @@
+package eu.esdihumboldt.hale.io.jdbc.test;
+
+import java.net.URI;
+
+
+public class DBImageParameters {
+	
+	/**
+	 * The database user name
+	 */
+	private String user ;
```

Comment:
If we now have a custom configuration for the Docker images it would be good if we could configure things like user, password and database also through the configuration file - as these are settings that are closely related to the Docker image itself, not the type of database.